index.js

```diff
@@ -10,7 +10,7 @@ export function initialState (props) {
 }
 
 export function render ({ props, state }, setState) {
-  const { items, onChange, removeOnHide } = props;
+  const { items, onChange, removeOnHide, customClass } = props;
```

Comment:
@kesla @iefserge im running into validation issues with semistandard when calling the prop `class` calling it anything else works fine.